packages/fx-core/src/plugins/resource/blazor/enum.ts

```diff
@@ -0,0 +1,34 @@
+export enum BlazorConfigKey {
+  /* Config from solution */
+  resourceGroupName = "resourceGroupName",
+  subscriptionId = "subscriptionId",
+  resourceNameSuffix = "resourceNameSuffix",
+  location = "location",
+  credential = "credential",
+  teamsAppName = "teamsAppName",
+
+  /* Config exported by Blazor plugin */
+  webAppName = "webAppName",
+  appServicePlanName = "appServicePlanName",
+  webAppEndpoint = "webAppEndpoint",
+  webAppId = "webAppId",
+
+  /* Intermediate */
+  site = "site",
+}
+
+export enum AppSettingsKey {
+  clientSecret = "CLIENT_SECRET",
+  clientId = "CLIENT_ID",
+  oauthHost = "OAUTH_AUTHORITY",
+  tabAppEndpoint = "TAB_APP_ENDPOINT",
+  aadMetadataAddress = "AAD_METADATA_ADDRESS",
+  botId = "BOT_ID",
+  botPassword = "BOT_PASSWORD",
```

Comment:
I see bot config here, which bot blazor plugin support here? Bot from Bot plugin by TS/JS or Bot in the Blazor plugin of .NET project?

Reply:
Bot of .NET project. The bot config works If we provision Azure Bot Service with bot plugin and host both bot code and blazor frontend code in the same Azure Web App.
Because we don't suppose to support bot in step 0, this is for future use.